apps/src/p5lab/redux/animationList.js

```diff
@@ -286,7 +286,8 @@ export function setInitialAnimationList(
   }
 
   // TODO (from 2020): Tear out this migration when it hasn't been used for at least 3 consecutive non-summer months.
-  if (spritesForV3Migration) {
+  // Migrate to v1 animation api for default sprites in SpriteLab
+  if (isSpriteLab && spritesForV3Migration) {
```

Comment:
We will either need to update `isSpriteLab` to also be true for its descendants (ie. Poetry + future spin-offs), or all add an additional boolean to check for Poetry.

Reply:
Any levels made after 2020, won't need to go through this migration - just SpriteLab projects from before 2020. So I think this logic can stay as-is for Poetry (circa 2021). Looking at `s/poetry-2021/lessons/1/levels/7` all the animations are `v1` animations.

Reply:
Oh, that makes sense!